packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -213,11 +213,14 @@ class ClayMultiSelect extends ClayComponent {
 
 		switch (event.data.char) {
 		case ',':
-			return this._handleItemAdded(
-				value,
-				this._getItemSchema(value, value),
-				event
-			);
+			if (this.allowInputCreateItem) {
+				return this._handleItemAdded(
+					value,
+					this._getItemSchema(value, value),
+					event
+				);
+			}
+			break;
```

Comment:
With this change you are no longer returning a boolean in the event that the creation is disallowed, but the JSDoc says that we return a bool.

Reply:
Oh yeah, you're correct about that. thanks.